pkgs/development/python-modules/nsz/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pycryptodome, enlighten, zstandard, kivy
+, withGUI ? true
+}:
+
+buildPythonPackage rec {
+  pname = "nsz";
+  version = "4.1.0";
+
+  src = fetchFromGitHub rec {
+    owner = "nicoboss";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-tdngXV+VUOAkg3lF2NOmw0mBeSEE+YpUfuKukTKcPnM=";
+  };
+
+  # doCheck = false;
```

Comment:
```suggestion
```

Reply:
That commented out `# doCheck = false;` should be removed.

Reply:
I'm impressed that I saw it, and still missed it. Should be done now.